mdm/orientdb-api/src/main/java/com/gentics/mesh/core/data/dao/MicroschemaDaoWrapper.java

```diff
@@ -1,10 +1,19 @@
 package com.gentics.mesh.core.data.dao;
 
+import com.gentics.mesh.core.data.root.MicroschemaRoot;
 import com.gentics.mesh.core.data.schema.HibMicroschema;
+import com.gentics.mesh.core.result.Result;
 
 /**
  * DAO for {@link HibMicroschema} operations.
  */
 public interface MicroschemaDaoWrapper extends PersistingMicroschemaDao {
 
+	/**
+	 * Return a list of all microschema container roots to which the microschema container was added.
+	 *
+	 * @return
+	 */
+	Result<? extends MicroschemaRoot> getRoots(HibMicroschema schema);
```

Comment:
Microschema's project retrieval has been improved, being aligned with the approach of Schema.